### ISSUE TITLE
Uncaught TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by mnaoumov** [OfficerHalf/obsidian-collapse-all#33]
```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at eval (plugin:obsidian-collapse-all-plugin:367:46)
    at Array.every (<anonymous>)
    at TagPaneProvider.tagsAreCollapsed (plugin:obsidian-collapse-all-plugin:367:22)
    at TagPaneProvider.allCollapsed (plugin:obsidian-collapse-all-plugin:355:21)
    at TagPaneProvider.updateButtonIcon (plugin:obsidian-collapse-all-plugin:208:39)
    at eval (plugin:obsidian-collapse-all-plugin:173:18)
    at Array.forEach (<anonymous>)
    at TagPaneProvider.updateButtonIcons (plugin:obsidian-collapse-all-plugin:172:21)
    at eval (plugin:obsidian-collapse-all-plugin:485:34)
    at Array.forEach (<anonymous>)
```

**Comment by OfficerHalf** [comment]
Can you give some context on where you see this?

**Comment by mnaoumov** [comment]
I didn't quite get when is that happening. I am currently debugging it. I found how to fix it already but now I am checking why did it happen at first place. Then I will send a PR

**Comment by OfficerHalf** [comment]
@mnaoumov I'm unable to reproduce the issue. In what scenario do you see this? Are you on an insiders version of Obsidian?

**Comment by mnaoumov** [comment]
Yes, I am on Insiders version 0.15.5

**Comment by OfficerHalf** [comment]
Released in 1.5.2. Thanks for the PR!
